### PR TITLE
Make WAF-specific variables/commands generic

### PR DESCRIPTION
**Pull request by indygriffiths**
Prep for an upcoming `salus` release

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -35,14 +35,14 @@
         content => "${cron_minute} ${cron_hour} * * * root cd /opt/salus/bin && ./salus 2>&1 | logger -t salus\n",
     }
 
-    # Set up the cron for uploading certificates to the WAF, if parameters are set
-    if $waf_cron_minute and $waf_cron_hour {
+    # Set up the cron for uploading certificates to the Dashboard, if parameters are set
+    if $cert_upload_cron_minute and $cert_upload_cron_hour {
         file { "/etc/cron.d/salus-waf":
```
Comment (ichaber) [inline manifests/install.pp]: Do you also want to replace `-waf` from `salus-waf` in the same way and call it `salus-cert`?

File: manifests/install.pp [hunk 2]
```diff
@@ -35,14 +35,14 @@
...
             ensure => "file",
             owner => "root",
             group => "root",
             mode => "0644",
-            content => "${waf_cron_minute} ${waf_cron_hour} * * * root cd /opt/salus/bin && ./salus wafcertificate:upload --all 2>&1 | logger -t salus\n",
+            content => "${cert_upload_cron_minute} ${cert_upload_cron_hour} * * * root cd /opt/salus/bin && ./salus certificate:upload --all 2>&1 | logger -t salus\n",
```
Comment (ichaber) [inline manifests/install.pp:45]: Am I missing something or does `certificate:upload` not exist yet?

File: manifests/install.pp (hunk 1 above)
Comment (indygriffiths) [reply manifests/install.pp]: Good call, renamed to `salus-upload`

File: manifests/install.pp (hunk 2 above)
Comment (indygriffiths) [reply manifests/install.pp:45]: Part of https://github.com/silverstripeltd/salus/pull/32

Comment (ichaber) [reply manifests/install.pp:45]: Merged that PR on salus before I merge that now. :)

**Pull request merged**